### ISSUE TITLE
QueryUtils' function `checkIsActuallyNumeric` throws exception if server enabled secure profile

### ISSUE THREAD

**Issue by satauros** [coldbox-modules/qb#112]
When executing queries that require the `checkIsActuallyNumeric` function to be executed on a server that enabled a secure profile (with the option **Disable access to internal ColdFusion Java components** checked), a Java method security exception is thrown.

The error is triggiered by the line of code on line no. 237: https://github.com/coldbox-modules/qb/blob/8994cd818bdbdf0774117a9797d919fe531124b1/models/Query/QueryUtils.cfc#L237

Possible fix for this method _could_ be altering the method as such:

```
private boolean function checkIsActuallyNumeric( required any value ) {
    return isNull( arguments.value ) || (
        isSimpleValue( arguments.value ) && arrayContainsNoCase(
            [
                "CFDouble",
                "Integer",
                "Double",
                "Float",
                "Long",
                "Short"
         ],
         listLast( getMetadata( arguments.value ), ' .' ) )
    );
}
```

**Comment by elpete** [comment]
If that passes all the tests, I’m fine with that. Want to make a PR?

On a separate note, I think that secure profile setting is problematic in more than one way. If there is not a good workaround, the only other option I would accept is to catch the security exception and return a varchar. In my opinion, though, enabling access is the better option.

**Comment by satauros** [comment]
I just ran all the tests and they were all successful:

![image](https://user-images.githubusercontent.com/12030878/86527440-03bcbe00-be9f-11ea-9982-8f59ade32652.png)

I'll create the pull request.

I just ran into this issue b/c on our production server we cannot enable the access to the java components, so I was forced to fix it another way. In my opinion, this is a fairly simple (and quick) workaround for those who are in the same situation and ran into the same issue. I am well aware that is probably an edge case, but I'm happy it can be fixed using this workaround.
